uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/FilteredSelectionDialog.java

```diff
@@ -53,7 +54,19 @@ public FilteredSelectionDialog waitForItems() {
 	}
 
 	public FilteredSelectionDialog selectItem(String text) {
-		new DefaultTable(this).getItem(text).select();
```

Comment:
Is it a bug in RedDeer or a missing feature? Why did you use your own implementation?

Reply:
I obtained error when I tried to select item, which was already selected on loaded wizard page... this problem occurs if there is only one item in table after query or if previous selection is remembered